src/test/fuzz/fuzz.cpp

```diff
@@ -22,7 +22,9 @@ static bool read_stdin(std::vector<uint8_t>& data)
     return length == 0;
 }
 
-static void initialize()
+// Default initialization: Override using a non-weak initialize().
+__attribute__((weak))
+void initialize()
 {
     const static auto verify_handle = MakeUnique<ECCVerifyHandle>();
```

Comment:
Yes, but I'd rather do that in a follow-up PR since some existing fuzzers rely on this. I want to keep this PR `Parse(…)` only to keep it simple. It will take some investigation to figure our which fuzzers that rely on the current behaviour.